app/views/user_mailer/assignment_default_notification.html.erb

```diff
@@ -0,0 +1,16 @@
+<% content_for :message do %>
+  <p><%= t(".defaulted_on_assignment.html",
+           offer_byline: style_bold(@assignment.offer_signup.pseud.byline),
+           request_byline: style_bold(@assignment.request_signup.pseud.byline),
```

Comment:
```suggestion
           offer_byline: style_bold(@assignment.offering_pseud.byline),
           request_byline: style_bold(@assignment.requesting_pseud.byline),
```
We do want the pinch hitter if there is one. We just don't want the `* (pinch hitter)` indication. Sorry for not being clear!